neuralprophet/utils_metrics.py

```diff
@@ -22,21 +22,21 @@ def get_metrics(metric_input):
 
     Returns
     -------
-        dict
-            Dict of torchmetrics.Metric metrics.
+        dict or list
```

Comment:
Maybe we should rename this function to `check_metrics`

Reply:
function changed, now we are still getting metrics (only metric names, but still getting the metrics)